app/helpers/submissions_helper.rb

```diff
@@ -42,12 +41,20 @@ def get_submissions_table_info(assignment, groupings)
       rescue RuntimeError
         g[:commit_date] = I18n.t('group_repo_missing')
       end
-      g[:marking_state] = get_grouping_marking_state(assignment, grouping)
-      g[:grace_credits_used] = get_grouping_grace_credits_used(grouping)
-      g[:final_grade] = get_grouping_final_grades(grouping)
-      g[:state] = get_grouping_state(grouping)
-      g[:section] = get_grouping_section(grouping)
-      g[:tags] = get_grouping_tags(grouping)
+      begin # if any thing raises an error, catch it and log in the object.
+        g[:class_name] = get_any_tr_attributes(grouping)
+        g[:repository] = get_grouping_repository(assignment, grouping)
+        g[:marking_state] = get_grouping_marking_state(assignment, grouping)
+        g[:grace_credits_used] = get_grouping_grace_credits_used(grouping)
+        g[:final_grade] = get_grouping_final_grades(grouping)
+        g[:state] = get_grouping_state(grouping)
+        g[:section] = get_grouping_section(grouping)
+        g[:tags] = get_grouping_tags(grouping)
+      rescue
+        g[:error] = true
```

Comment:
At the very least we should log the grouping and the error that was raised. (MarkUsLogger)